src/main/java/com/fasterxml/jackson/core/io/ContentReference.java

```diff
@@ -203,10 +249,11 @@ public Object getRawContent() {
      * which content is counted, either bytes or chars) to use for truncation
      * (so as not to include full content for humongous sources or targets)
      *
+     * @see ErrorReportConfiguration#builder()#_maxRawContentLength
      * @return Maximum content snippet to include before truncating
      */
     protected int maxContentSnippetLength() {
```

Comment:
Let's go with straight change since it is internal (option (1)). I feel confident this won't break usage by others, given it is reasonable recently added internal method.